### PR TITLE
Increased Timeout on Service sender to 20 seconds from 1 second.

### PR DESCRIPTION
**Pull request by cstait**

### CODE REVIEW

**Comment by mattgwagner** [comment]
So, the context here was that I wanted to fail fast -- if it's taking longer than one second to pipe over to the notification service, then we might as well have waited for the SMTP server. And if the notification service is taking that long to ack the item, then we're probably doing something wrong. Just offering :-)

**Comment by jerbri** [comment]
@cstait this was supposed to be on the client side, not in the common library.

@mattgwagner haha, yes. Unfortunately, it's failing fast -- and failing often...he's trying to figure out if it's the service or the store or what.  Welcome back stateside btw :)

**Comment by mattgwagner** [comment]
Interesting -- in theory, the service should just accept the DTO and pipe it into the Redis list.

I would advise checking the service logs to see how long it's taking to process the message. Also perhaps check the Redis memory usage, it probably has a lot of data hanging around in the processed messages list.

File: CertiPay.Common.Notifications/Notifications/ServiceSender.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class ServiceSender : INotificationSender<EmailNotification>, INotificati
 
         public readonly Uri ServiceUri;
 
-        private static TimeSpan Timeout => TimeSpan.FromSeconds(1);
```
Comment (mattgwagner) [inline CertiPay.Common.Notifications/Notifications/ServiceSender.cs]: Another option is to make this a public property that can be modified by the calling client.

Comment (cstait) [reply CertiPay.Common.Notifications/Notifications/ServiceSender.cs]: Yup, that route would better suit our needs. :+1:

**Pull request closed without merging**